articles/152_roslaunch_multi_machine_launch.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: default
+title: ROS 2 Multi-Machine Launching
+permalink: articles/roslaunch_mml.html
+abstract:
+  Robotic systems are often distributed across multiple networked machines.
+  This document describes proposed modifications and enhancements to ROS2's
+  launch system to facilitate launching, monitoring, and shutting
+  down systems spread across multiple machines.
+author: '[Matt Lanting](https://github.com/mlanting)'
+published: false
+---
+
+- This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Authors: {{ page.author }}
+
+## Purpose
+
+Allow a system of ROS nodes to be launched on a hardware architecture that is
+spread across multiple networked computers and facilitate introspection and
+management of the system from a single machine.
+
+## Justification
+
+Nodes can need to run on different hosts for a variety of reasons.  Some possible
```

Comment:
@mlanting `can need`? would `may need` sound better?

Reply:
"May need" sounds better to me